framework/13-inch/12th-gen-intel/default.nix

```diff
@@ -11,6 +11,13 @@
       # same as 13th gen framework 13-inch
       hardware.framework.laptop13.audioEnhancement.rawDeviceName = lib.mkDefault "alsa_output.pci-0000_00_1f.3.analog-stereo";
     }
+    {
+      # fix hdmi audio
+      # src https://community.frame.work/t/hdmi-audio-output/9523/23
+      boot.extraModprobeConfig = ''
+        options snd-intel-dspcfg dsp_driver=3
```

Comment:
Do both 12th and 13th gen intel framework laptops have similar chipsets for sound? The thread is quite old but not sure about the resolution.